etc/config/wasm.defaults.properties

```diff
@@ -0,0 +1,21 @@
+compilers=&wasmtime
+objdumper=llvm-objdump
+objdumperType=llvm
+binaryHideFuncRe=^.*::(:?native_to_wasm_trampoline|wasm_to_native_trampoline|array_to_wasm_trampoline).*$
+
+group.wasmtime.compilers=wasmtime2001
+group.wasmtime.compilerType=wasmtime
+group.wasmtime.options=-Ddebug-info=y -Ccache=n
+group.wasmtime.supportsBinary=true
+group.wasmtime.supportsBinaryObject=true
+group.wasmtime.versionFlag=--version
+group.wasmtime.isSemVer=true
+
+compiler.wasmtime2001.name=Wasmtime 20.0.1
+compiler.wasmtime2001.exe=/opt/compiler-explorer/wasmtime-20.0.1/wasmtime
```

Comment:
The path in the `defaults` environment should be the one where a typical wasm installation would find the binary - this is usually `/usr/bin/{binaryname}` for most other language compilers, but whatever works best in wasm's case would be ok

Reply:
I'm not too sure what to do here, wasmtime defaults to installing via a shell script, and the default location is `$HOME/.wasmtime/bin/wasmtime`, installing via `cargo` (the rust package manager) also installs it in the users home directory. I also don't think there are any system packages for it. At least that I could find.

Reply:
Hmm, I see. We could probably merge as-is then, but let's see what others think first, I'll bring it up tomorrow if noone checks this before :)